### ISSUE TITLE
How much time did you spend on testing your model?

### ISSUE THREAD

**Issue by hjynwa** [divelab/Spatial-VAE-via-MVND#1]
Hi, I try to use parzen window to test my model, but it take too much time to compute the final result. So I want to know did you also spend much time on testing?
I use GTX Titan X for computing.
Thanks!

**Comment by Nate1874** [comment]
Yes, the parzen window is quite slow, may takes hours. Depends on the task and data.

**Comment by hjynwa** [comment]
Hi, thanks for your answer. Now I got the likelihood "ll" and standard error "se", but the "ll" is negative numbers, such as "-15762496", is it reasonable? May I know the "ll" and "se" of your model ?
Thanks so much!

**Comment by Nate1874** [comment]
We do not have the parzen window evaluation on celeba data but your number "-15762496" looks not reasonable.
